src/Hosting/WindowsService/src/Microsoft.Extensions.Hosting.WindowsService.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>.NET hosting infrastructure for Windows Services.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>hosting</PackageTags>
+    <IsShipping>true</IsShipping>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.Hosting.Abstractions" />
+    <Reference Include="System.ServiceProcess.ServiceController" />
```

Comment:
What happens on non-windows systems? When I try to run the tests I see a PlatformNotSupportedException when the ServiceBase constructor is called.

Reply:
The tests are marked with OSSkipCondition's. Any idea why those wouldn't be working?